excel-importer-test/src_test/com/axonivy/util/excel/test/TestExcelReader.java

```diff
@@ -22,12 +23,12 @@ void parseColumns_xlsx(@TempDir Path dir) throws IOException {
     TstRes.loadTo(path, "sample.xlsx");
     Workbook wb = ExcelLoader.load(path);
     List<Column> columns = ExcelReader.parseColumns(wb.getSheetAt(0));
-    assertThat(columns).extracting(Column::name)
-      .contains("Firstname", "Lastname");
+    assertThat(columns).extracting(Column::getName)
+        .contains("firstname", "lastname");
```

Comment:
looks as if we have lost the human readable name, upper and lower case chars?
in german many words are written uppercase, so we frequently have such columns and users would be unfamiliar with 'lowercased' column names.
... can we keep the column name "FirstName", "ZIP", "SuperUser" and the like?